catchall_server.py

```diff
@@ -1,15 +1,31 @@
-from flask import Flask
+from quart import Quart
+from yagna_connector import YagnaConnector
+from serializable_request import Request
+
+#   TODO: do we want to use hypercorn?
+import hypercorn
+hypercorn.Config.startup_timeout = 999
```

Comment:
1. In general, yes
2. From the http-requestor-proxy POV, we might want to have a hardcoded "hypercorn + quart" combo, instead of allowing user to handle serving our ASGI catchall_server.py, because of issues like this that are non-obvious and have to be handled correctly (there might be more, e.g. how signals are captured and distributed between Executor/workers, shutdown, or how workers initialization is entangled with Executor initialization)
3. --> If we go with hypercorn+quart, this should still be outside of app, in a simple file looking like this:
https://pgjones.gitlab.io/hypercorn/how_to_guides/api_usage.html and `hypercorn.Config.startup_timeout` would go there
4. As long as we are prototyping, I think it is OK to have this here, but I'll change that if you insist.